### PR TITLE
proxmox-auto-install-assistant: init at 8.2.6

### PR DESCRIPTION
**Pull request by alcelafranque**
## Add proxmox custom installation image creation package

This tool lets you create proxmox ISOs for automatic installation.
[Automated_Installation Assistant_Tool](https://pve.proxmox.com/wiki/Automated_Installation#Assistant_Tool)

- Built on platform(s)
  - [x] x86_64-linux

- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = true`
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)

### CODE REVIEW

File: pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook makeWrapper ];
+  buildInputs = [
+    dpkg
+    libgcc
+    xorriso
+  ];
+
+  unpackPhase = ''
```
Comment (Aleksanaa) [inline pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix:28]: Not needed? It should unpack automatically (in current dir), and the remaining work should be done in `installPhase`

File: pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    dpkg-deb -x $src $out
+  '';
+
+  installPhase = ''
```
Comment (Aleksanaa) [inline pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix:32]: `runHook preInstall` and `runHook postInstall`

File: pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  postFixup = ''
+    wrapProgram $out/bin/proxmox-auto-install-assistant \
+      --prefix PATH : ${lib.makeBinPath [ xorriso ]}
+  '';
+
+  meta = with lib; {
+    description = "Proxmox Auto Install Assistant";
+    homepage = "https://www.proxmox.com";
+    license = licenses.agpl3Plus;
+    maintainers = with maintainers; [ alafranque ];
```
Comment (Aleksanaa) [inline pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix:46]:
```suggestion
    maintainers = with maintainers; [ alafranque ];
    mainProgram = "proxmox-auto-install-assistant";
    platforms = [ "x86_64-linux" ];
    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
```

File: pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix
Comment (Aleksanaa) [inline pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix:1]: Anyway, is it possible to build from source?

File: pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  stdenv,
+  fetchurl,
+  dpkg,
+  autoPatchelfHook,
+  libgcc,
+  xorriso,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation rec {
```
Comment (Aleksanaa) [inline pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix:12]:
```suggestion
stdenvNoCC.mkDerivation rec {
```

File: pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  pname = "proxmox-auto-install-assistant";
+  version = "8.2.6";
+
+  src = fetchurl {
+    url = "http://download.proxmox.com/debian/pve/dists/bookworm/pve-no-subscription/binary-amd64/proxmox-auto-install-assistant_${version}_amd64.deb";
```
Comment (SuperSandro2000) [inline pkgs/by-name/pr/proxmox-auto-install-assistant/package.nix:17]: They surely have a https download link

**Comment by alcelafranque** [comment]
Thanks for your reviews :)

I'm going to look at doing everything with a build of the source code directly next WE.

**Comment by emilylange** [comment]
Superseded by #348351

**Pull request closed without merging**